polar-language-server/src/lib.rs

```diff
@@ -141,61 +165,81 @@ impl PolarLanguageServer {
             // Nothing to do when we receive the `Initialized` notification.
             Initialized::METHOD => (),
 
-            _ => log(&format!("on_notification {} {:?}", method, params)),
+            _ => log("unexpected notification"),
```

Comment:
Super nit: potentially worth logging the method + params still for debugging purposes. Good change to the message though

Reply:
The method is now logged out here: https://github.com/osohq/oso/blob/9c0400beecd94697e36923e5f29d78fc4ca0ae86/polar-language-server/src/lib.rs#L56

I figured debug-printing the params would be noisy, but I guess it doesn't really matter. We should *never* hit this unless we're adding new LSP functionality. I can add the params back.

Reply:
Cool, that makes sense. (happy with/without params tbh)